installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc

```diff
@@ -12,6 +12,24 @@ toc::[]
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You have read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 
+[id="planning-a-bare-metal-cluster-for-ocp-virt"]
+== Planning a bare metal cluster for {VirtProductName}
+
+If you will use {VirtProductName}, it is important to be aware of several requirements before you install your bare metal cluster.
+
+* You must have multiple worker nodes at the time of cluster installation. The high availability (HA) capability flag is permanently set when a cluster is first installed. If you do not have multiple worker nodes, you cannot use live migration.
```

Comment:
I think this is sufficient to convey the logic, but could be somewhat clearer because you don't explicitly link HA and live migration. In my mind:

If you wish to use live migration features in {VirtProductName}, you must have multiple worker nodes at the time of cluster installation.

1. Live migration in {VirtProductName} requires the cluster-level high availability (HA) capability flag.
2. The HA flag is set at cluster install time and cannot be changed afterwards.
3. If there are less than two worker nodes defined at cluster install time, the HA flag will be set to false.